libraries/api.rb

```diff
@@ -80,6 +101,29 @@ def cv_fs_filter(endpoint, cv_token, subclient_name, filters)
       end
     end
 
+    def cv_fs_subclient_has_plan(endpoint, cv_token, subclient_name)
+      props = _cv_fs_properties(endpoint, cv_token, subclient_name)
+      raise 'Received incorrect output from CommVault API' unless props
+      if props['subClientProperties'][0].key?('planEntity')
+        Chef::Log.debug "Plan entity: #{props['subClientProperties'][0]['planEntity']}"
+      else
+        Chef::Log.debug 'planEntity not set'
+      end
+      props['subClientProperties'][0].key?('planEntity') && props['subClientProperties'][0]['planEntity'].key?('planName')
+    end
+
+    def cv_fs_subclient_assign_plan(endpoint, cv_token, subclient_name, plan_name)
+      unless subclient_name == 'default'
+        raise 'We are unable to manage any subclient other than default for now'
+      end
```

Comment:
```suggestion
      raise 'We are unable to manage any subclient other than default for now' unless subclient_name == 'default'

```

Just a preference thing, but if it's short enough for one line it's preferred, and to leave an extra line to make it clear that you're going to return or raise an exception.

Reply:
Done